CHANGES.rst

```diff
@@ -47,10 +47,10 @@ Features added
 * #13065: Enable colour by default in when running on CI.
   Patch by Adam Turner.
 * Allow supressing warnings from the :rst:dir:`toctree` directive when a glob
-  pattern doesn't match any documents, via the new ``toc.empty_glob``
+  pattern doesn't match any documents, via the new ``toc.glob_not_matching``
   warning sub-type.
   Patch by Slawek Figiel.
-* #9732: Add the new ``autodoc.mocked_object`` warnings sub-type.
+* #9732: Add the new ``autodoc.mock_objects`` warnings sub-type.
```

Comment:
?

Reply:
Strange. That's of course unintended.. I possibly  have messed up git when bringing in the remote upstream merge.

Reply:
Ok fixed. I'm struggling with the merge-main-into-branch workflow. I'm much more comfortable with rebases. Sorry for the confusion.

Reply:
No worries -- I tend to use the merge interface in PyCharm for tricky merges, which simplifies things greatly.